include/slate/BaseMatrix.hh

```diff
@@ -2098,9 +2098,11 @@ void BaseMatrix<scalar_t>::listBcastMT(
     using BcastTag =
         std::tuple< int64_t, int64_t, std::list<BaseMatrix<scalar_t> >, int64_t >;
 
-    #pragma omp taskloop slate_omp_default_none \
-        shared( bcast_list ) \
-        firstprivate(life_factor, layout, mpi_size, is_shared)
+    #if defined( SLATE_HAVE_listBcastMT )
```

Comment:
As discussed, let's change this to `SLATE_HAVE_MT_BCAST`, since listBcastMT is a rather internal name, and document that option in INSTALL.md. A user would have to set
`CXXFLAGS=-DSLATE_HAVE_MT_BCAST`, correct?

Reply:
I changed it.
Correct, we need `CXXFLAGS=-DSLATE_HAVE_MT_BCAST` to enable mulit-threading broadcast.